### PR TITLE
Add explanatory comment

### PR DESCRIPTION
**Pull request by amcasey**
FYI @rchande @dpoeschl

### CODE REVIEW

**Comment by dpoeschl** [comment]
Should a note be on ISignature for API consumers? Should we deprecate this property?

**Comment by dpoeschl** [comment]
:+1: regardless, doing the rest of that can be @rchande's problem. :smile:

File: src/EditorFeatures/Core/Implementation/IntelliSense/SignatureHelp/Presentation/Signature.cs [hunk 1]
```diff
@@ -68,6 +68,9 @@ public IParameter CurrentParameter
             }
         }
 
+        /// <remarks>
+        /// The documentation is included in <see cref="Content"/> so that it will be formatted.
+        /// </remarks>
```
Comment (rchande) [inline src/EditorFeatures/Core/Implementation/IntelliSense/SignatureHelp/Presentation/Signature.cs]: Nit: classified, not formatted. (Hangs head in shame)

**Comment by rchande** [comment]
:+1:

**Comment by amcasey** [comment]
Closing to fix branch mistake.

**Pull request closed without merging**